db/schema_tables.cc

```diff
@@ -837,13 +837,14 @@ future<table_schema_version> calculate_schema_digest(distributed<service::storag
     return calculate_schema_digest(proxy, features, std::not_fn(&is_system_keyspace));
 }
 
```

Comment:
>schema_tables: convert_schema_to_mutations: return mutations rather t…
>…han canonical_mutations

>Let the caller make the canonical mutations.
>The next patch will do that gently in a thread.

missing motivation

Reply:
the above is supposed to be the motivation - i.e. to let the caller make the canonical mutation in a thread.
I'll rephrase it to clarify the intent.